core/src/main/java/co/aikar/commands/contexts/IssuerAwareContextResolver.java

```diff
@@ -26,5 +26,13 @@
 import co.aikar.commands.CommandExecutionContext;
 import co.aikar.commands.CommandIssuer;
 
-
-public interface IssuerAwareContextResolver<T, C extends CommandExecutionContext<?, ? extends CommandIssuer>> extends ContextResolver<T, C> {}
+/**
+ * The very same as {@link ContextResolver} except it now also must know the issuer.
```

Comment:
Wrong wording. 
ISsuer aware resolvers may conditionally consume input arguments, and might resolve the context purely based on the issuer of the command rather than command input.

Examples would be "What world are you standing in? What Clan are you currently in? What is your current chat channel?"